### PR TITLE
Add more aliases

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
💅: "nail_care", "slay"
🍮: "pudding", "flan"
💀: "dead", "ong"
😐: "bruh"
😒: "ugh"
🤨: "suspicious"
😭: "sobbing"
👋: "wave"
🍙: "onigiri", "omusubi"
🍘: "senbei"
⚡: "zap"
🍢: "skewer"
💯: "100"
🏳️‍🌈: remove inapplicable aliases, as those other terms have their own flags.
🏳️‍⚧️: "transgender", "flag", "pride" 
1️⃣2️⃣3️⃣4️⃣5️⃣6️⃣7️⃣8️⃣9️⃣🔟0️⃣: "one", "two", "three", etc...
🔠: "letters", "uppercase"
🔡: "letters", "lowercase"

### CODE REVIEW

**Comment by ThatOneCalculator** [comment]
@muan any chance of getting this merged?

File: dist/emoji-en-US.json [hunk 1]
```diff
@@ -9887,12 +9921,8 @@
     "pride",
     "gay",
     "lgbt",
-    "glbt",
     "queer",
-    "homosexual",
-    "lesbian",
```
Comment (muan) [inline dist/emoji-en-US.json:9893]: why was this removed?

File: dist/emoji-en-US.json [hunk 2]
```diff
@@ -295,14 +295,16 @@
...
+    "suspicious"
   ],
   "😐": [
     "neutral_face",
     "indifference",
     "meh",
     ":|",
-    "neutral"
+    "neutral",
+    "bruh"
```
Comment (muan) [inline dist/emoji-en-US.json:307]: I am not sure about these slangy ones. @JoshuaKGoldberg any thoughts?

File: dist/emoji-en-US.json [hunk 3]
```diff
@@ -738,7 +741,9 @@
     "dead",
     "skeleton",
     "creepy",
-    "death"
+    "death",
+    "dead",
+    "ong"
```
Comment (muan) [inline dist/emoji-en-US.json:746]: this one as well

File: dist/emoji-en-US.json [hunk 4]
```diff
@@ -4598,12 +4607,15 @@
     "rice_cracker",
     "food",
     "japanese",
-    "snack"
+    "snack",
+    "senbei"
```
Comment (muan) [inline dist/emoji-en-US.json:4611]: this one too

**Review by muan — COMMENTED** [review]
Sorry I think this just slipped through my inbox.

File: dist/emoji-en-US.json (hunk 1 above)
Comment (ThatOneCalculator) [reply dist/emoji-en-US.json:9893]: - "glbt" is not a term that's used in the queer community, and is used as a dogwhistle to downplay women's rights. https://theforeword.org/832/editorials/the-l-in-lgbt-and-why-order-matters/ 
- the pride flag is not the same as the gay man flag or the lesbian flag.

File: dist/emoji-en-US.json (hunk 4 above)
Comment (ThatOneCalculator) [reply dist/emoji-en-US.json:4611]: "senbei" is just the japanese name for 🍘

File: dist/emoji-en-US.json (hunk 1 above)
Comment (muan) [reply dist/emoji-en-US.json:9893]: Sorry, I just meant lesbian specifically. Is it in the same vein that gay is preserved but not lesbian?

Comment (ThatOneCalculator) [reply dist/emoji-en-US.json:9893]: Ah sorry I missed this reply. Generally, 🏳️‍🌈 is referred to as just the pride flag (although it started out as the gay flag). Although queer/LGBTQ pride obviously includes lesbian, the 🏳️‍🌈 flag isn't the lesbian flag and thus shouldn't be aliased to "lesbian".

Comment (muan) [reply dist/emoji-en-US.json:9893]: considering that there’s not a lesbian flag in emoji, do you mind if we keep this?

Comment (ThatOneCalculator) [reply dist/emoji-en-US.json:9893]: The lesbian flag is often implemented w/ a ZWJ or custom emoji.

Comment (muan) [reply dist/emoji-en-US.json:9893]: IMO given the fact that neither gay man flag and nor lesbian flag is in unicode, I think erring on being inclusive of the broader definition for pride is my preference so that there is at least something to be found.

**Comment by muan** [comment]
Thanks! Released with https://github.com/muan/emojilib/releases/tag/v3.0.11.
